base/public.jl

```diff
@@ -71,6 +71,9 @@ public
     isoperator,
     isunaryoperator,
 
+# Integer math
+    mul_hi,
```

Comment:
If this is going to be public, it'd be nice for it to have a more descriptive name (and ideally one that is stylistically consistent, i.e. germanic case rather than snake case). Perhaps `mulhighhalf` or something like that?

Reply:
Eh, I think the name `mul_hi` seems ok.